docs/workload-development-kit/fabric-data-plane.md

```diff
@@ -12,41 +12,98 @@ ms.date: 05/21/2024
+More details on how to obtain a token can be found in the [Microsoft Fabric Workload Development REST API documentation](https://go.microsoft.com/fwlink/?linkid=2271986).
 
-For examples of token authentication, please refer to the Microsoft Fabric Developer kit sample.
+### Read Metadata
 
-More details on how to obtain a token can be found in the [Microsoft Fabric Workload Development REST API documentation](https://go.microsoft.com/fwlink/?linkid=2271986). 
+Fabric REST APIs also provide you with a way to access the different properties of items. For example, querying the [Get Lakehouse API](/rest/api/fabric/lakehouse/items/get-lakehouse?tabs=HTTP) provides you with the metadata for a certain Lakehouse, including useful properties such as OneLake paths and the SQL connection string.
+Another useful endpoint is the [Get Warehouse API](/rest/api/fabric/warehouse/items/get-warehouse?tabs=HTTP), which returns the following information:
+
+```
+    {
+        Id: Guid
+        Type: string
+        DisplayName: string
+        Description: string
+        WorkspaceId: Guid
+        Properties {
+            ConnectionInfo: string
+            CreatedDate: DateTime
+            LastUpdatedTime: DateTime
+        }
+    }
+```
+
+Here, the "ConnectionInfo" property is the FQDN (Fully Qualified Domain Name) of the Warehouse SQL Server. With this FQDN, we'll be able to establish a SQL connection. For more information, see [Connectivity to Data Warehousing in Microsoft Fabric](../data-warehouse/connectivity.md).
+For implementation examples, refer to the [Microsoft Fabric Workload Development Kit](./index.yml).
 
 ### Reading data
 
 Once you authenticate, you can connect to OneLake using [Azure Data Lake Storage REST APIs](/rest/api/storageservices/data-lake-storage-gen2) to read different types of data. We recommend utilizing the [Delta Lake protocol](https://github.com/delta-io/delta/blob/master/PROTOCOL.md) in order to read tables.
 
+Alternatively, if you choose to utilize Azure SQL Database you can implement the following instructions in order to read data from a Warehouse.
+
+1. Create an authorization context. For an example of creating an authorization context, see the [AuthenticateDataPlaneCall method](https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/blob/main/Backend/src/Services/AuthenticationService.cs).
+1. Acquire a token with the Warehouse.Read.All scope on behalf of the user using the bearer token passed from the Frontend.
+1. Using the "Fabric" token, call the "Get Warehouse" API that was previously referenced.
```

Comment:
"Fabric" --> *Fabric*

'"Get Warehouse"' --> Get Warehouse (and add the link).